src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
```

Comment:
use try and catch 
```suggestion
export async function getComments(postId: number) {
     try {
    const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
    return response.json();
  } catch (e) {
     throw new Error(`${response.status} - ${response.statusText}`);
  }
}
```

as you use same error handler in few places you can move it to separate function to api.ts and import to every file, in which you need to use it